helm/kanister-operator/values.yaml

```diff
@@ -2,8 +2,8 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 image:
-  repository: kanisterio/controller
-  tag: 0.20.0
+  repository: tdmanville/kanister-controller
```

Comment:
This looks like a local testing change

Reply:
it is indeed - though we do need to change this to `kanisterio/controller`